charts/newrelic-logging/values.yaml

```diff
@@ -37,6 +37,69 @@ fluentBit:
   # - name: HTTPS_PROXY
   #   value: http://example.com:3128
 
+  # New Relic default configuration for fluent-bit.conf (service, inputs, filters, outputs)
+  # and parsers.conf (parsers). The configuration below is not configured for lowDataMode and will
+  # send all attributes.  If custom configuration is required, update these variables.
+  config:
+    service: |
+      [SERVICE]
+          Flush         1
+          Log_Level     ${LOG_LEVEL}
+          Daemon        off
+          Parsers_File  parsers.conf
+          HTTP_Server   On
+          HTTP_Listen   0.0.0.0
+          HTTP_Port     2020
+
+    inputs: |
+      [INPUT]
+          Name              tail
+          Tag               kube.*
+          Path              ${PATH}
+          Parser            ${LOG_PARSER}
+          DB                ${FB_DB}
+          Mem_Buf_Limit     7MB
+          Skip_Long_Lines   On
+          Refresh_Interval  10
+
+    filters: |
+      [FILTER]
+          Name           kubernetes
+          Match          kube.*
+          # We need the full DNS suffix as Windows only supports resolving names with this suffix
+          # See: https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#dns-limitations
+          Kube_URL       https://kubernetes.default.svc.cluster.local:443
+          K8S-Logging.Exclude ${K8S_LOGGING_EXCLUDE}
+
+      [FILTER]
+          Name           record_modifier
+          Match          *
+          Record         cluster_name ${CLUSTER_NAME}
+          Record         k8s.cluster.name ${CLUSTER_NAME}
```

Comment:
💭 Hey, any reason we add `k8s.cluster.name` record with same value as `cluster_name`?